components/PostForm/PostForm.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+export function Postform() {
+  function handleClick(event) {
+    event.preventDefault();
+    document.getElementById("display").innerHTML =
+      document.getElementById("inputText").value;
+  }
+
+  return (
+    <Post>
+      <div>
+        <label>Post</label>
+        <input type="text" id="inputText" required name="text" />
```

Comment:
Make sure to connect the label to the input element using the `for` attribute (in react that's `htmlFor`).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

```suggestion
        <label htmlFor="inputText">Post</label>
        <input type="text" id="inputText" required name="text" />
```

Reply:
That way, clicking on the label will focus the input element.